src/flows/ContractAmendment/hooks.ts

```diff
@@ -85,9 +85,13 @@ const useContractAmendmentSchemaQuery = ({
         jsfSchema = schema;
       }
       const copyFieldValues = { ...fieldValues };
-      return createHeadlessForm(jsfSchema, {
-        initialValues: copyFieldValues || buildInitialValues(employment),
+      const hasFieldValues = Object.keys(copyFieldValues).length > 0;
+      const result = createHeadlessForm(jsfSchema, {
+        initialValues: hasFieldValues
+          ? copyFieldValues
+          : buildInitialValues(employment),
```

Comment:
Adding a bug fix to this PR. `buildInitialValues` was never used in the first render because `copyFields` is a `{}` by default and not a falsy value.